### ISSUE TITLE
About the data

### ISSUE THREAD

**Issue by cyclopssssss** [Tlili-ahmed/2BiVQA#15]
Hello, i want to ask about the data when i use KoNVid as the dataset, 
1.in your End2End_train.py, the data is sorted and the first 80% is used as the training set, and the last 20% is used as the validation set, however, in your test_model.py, x_test_konvid.npy and y_test_konvid.npy don't look likes the same split,
2.In the y_test_konvid.npy, the values of the labels seem to be between 0 and 1, while the original labels are between 1.22 and 4.64. Have these data undergone some preprocessing?
I would be very grateful if you could answer my doubts！

**Comment by Ahmed-Telili** [comment]
Hi @cyclopssssss, 

Sorry for the late reply!

The dataset split should be the same if you generated x_test_konvid.npy and y_test_konvid.npy using the provided scripts. However, if the dataset has been updated or modified since 2021 (when these files were created), there might be slight differences.

The labels in y_test_konvid.npy are normalized by dividing the original values by 5. This transformation scales them to the range [0,1].

Let me know if you need any clarifications!
